command/file_manipulation.pm

```diff
@@ -1116,6 +1116,41 @@ sub Run ($)
             mkdir( $filename );
         }
     }
+    ### type=  chdir
+    ################
+    elsif ($type eq "chdir") {
+        if (defined $filename) {
+            if (-d $filename ) {
+                $current_dir = $filename;
+            } else {
+                print STDERR __FILE__, ":\n",
+                    "  Could not chdir into $filename, it is not a directory!\n";
+                return 0;
+            }
+        } else {
+            $current_dir = main::GetVariable ('root');
+        }
+    }
+
+    ### type=  rmmkcd
+    # "rm, mkdir, chdir"
+    # Create or re-create and change into a directory in one command.
+    # Created for CMake build directories.
+    ################
+    elsif ($type eq "rmmkcd") {
+        if (-e $filename) {
+            rmtree ($filename, 0, 0);
+        }
+        mkdir ($filename);
+        if (-d $filename ) {
+            $current_dir = $filename;
+        } else {
+            print STDERR __FILE__, ":\n",
+                "  Could not chdir into $filename, it is not a directory!\n";
```

Comment:
Should this be an error about `mkdir`?
Is there a missing `chdir` in the sequence?

Reply:
There is a preexisting `chdir` at the end of the function that this command and the `chdir` command are using.

I could also remove this command, since I'm not planning on using it anymore.